project/src/main/creature-library.gd

```diff
@@ -0,0 +1,40 @@
+class_name CreatureLibrary
```

Comment:
Add comments.

Reply:
Added comments, `Manages data for all creatures in the world, including the player.`